packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -119,3 +112,10 @@ The content in this example is placed in a card to better illustrate how the sec
 
 ```ts file="./PageCenteredSection.tsx"
 ```
+
+### Deprecated page header
```

Comment:
Can we move this example into it's own file in the deprecated folder along with the PageHeader.  That way it would show up under a deprecated tab.  That way the non recommend solution is not shown on the main `Page` page.
cc @mcarrano what do you think?

Reply:
@tlabaj I think so, but let me make sure I understand. Are you saying that when the user goes to the Page component, they will then see two React tabs? One with the masthead examples and another called "React deprecated" that has the old example using the PageHeader? I guess that's OK.

Reply:
Yes. That is what I am saying.  This one example using the PageHeader would be in the `React deprecated` tab.

Reply:
Then yes, I think that works.